backend/src/metrics.rs

```diff
@@ -200,6 +201,12 @@ impl MemInfo {
 
         Some(out)
     }
+
+    // on non-linux systems we don't gather any memory info
```

Comment:
We start all comments with an uppercase letter. That also helps us detect when we forgot to remove commented-out code as a linter can simply check if a comment start with a lower case letter. So:

```suggestion
    // On non-linux systems we don't gather any memory info.
```

(I personally also like to end all comments that are a real sentence with a period)